modules/tinymce/src/core/test/ts/browser/paste/PasteStylesTest.ts

```diff
@@ -67,4 +67,13 @@ describe('browser.tinymce.core.paste.PasteStylesTest', () => {
     Clipboard.pasteItems(TinyDom.body(editor), { 'text/html': '<span style="color: rgb(224, 62, 45);">b</span>' });
     TinyAssertions.assertContent(editor, `<p><span style="color: rgb(224, 62, 45);">b</span></p>`);
   });
+
+  it('TINY-8525: paste span without any color styles, paste_webkit_styles: color,font-family', () => {
+    const editor = hook.editor();
+    editor.options.set('paste_webkit_styles', 'color,font-family');
```

Comment:
Should we unset this at the end of the test?

Reply:
There's a hook that does that: https://github.com/tinymce/tinymce/blob/feature/TINY-8525/modules/tinymce/src/core/test/ts/browser/paste/PasteStylesTest.ts#L23. Though looking at that it's got the wrong option name 🤦

Reply:
Have fixed the incorrect option name

Reply:
Ah right 👍